resources/run_sampling.rb

```diff
@@ -219,8 +219,10 @@ def _sample_probability_distribution(prob_dist, num_samples)
     end
 
     return_samples = {}
+    return_samples_v2 = {}
```

Comment:
New version.
Once the new version is accepted to be okay, we can remove the old version. Keeping both for now to make it possible to compare.